### PR TITLE
Polish Translation Update 2

### PR DESCRIPTION
**Pull request by szpigozzzzz**
(edited it because i saw the changes and i did it right)

this update is for prepearation for update 0.13.1

## Proposed Changes

  -added translation for partial dl wolf
  -added translation for the new advancement

### CODE REVIEW

**Comment by szpigozzzzz** [comment]
im sorry if i messed something up

**Comment by spacedan228** [comment]
ok, actually writing something in "Proposed Changes" is optional

**Comment by szpigozzzzz** [comment]
understood,i just got used to do that

File: src/main/resources/assets/changed/lang/pl_pl.json [hunk 1]
```diff
@@ -486,6 +487,10 @@
   "changed:advancements.main.tame_a_latex.title": "Niemożliwe stało się możliwym",
   "changed:advancements.main.tame_a_latex.description": "oswój latexową bestię",
 
+  "changed:advancements.main.kill_all_latex.title": "Srebro na potwory,a łom na latex",
+
```
Comment (LtxProgrammer) [inline src/main/resources/assets/changed/lang/pl_pl.json:491]: Small issue, there doesn't need to be a space here between the lines.

**Review by LtxProgrammer — CHANGES_REQUESTED** [review]

**Pull request merged**